### PR TITLE
pages.nl/*: add and update pages

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.nl/common/hping3.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# hping3
+
+> Geavanceerd pinghulpprogramma dat protocollen ondersteunt zoals TCP, UDP en IP.
+> Best run with elevated privileges.
```
Comment (gutjuri) [inline pages.nl/common/hping3.md:4]: This is not translated yet

File: pages.nl/windows/new-item.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# New-Item
+
+> Maak een nieuw bestand, directory, symbolische link of een registerinvoer.
+> This command can only be used through PowerShell.
```
Comment (gutjuri) [inline pages.nl/windows/new-item.md:4]: Not translated

**Review by gutjuri — APPROVED** [review]

File: pages.nl/common/linode-cli.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # linode-cli
 
 > Beheer Linode cloud-diensten.
-> Sommige subcommandos zoals `events` hebben hun eigen documentatie.
+> Sommige subcommando's zoals `events` hebben hun eigen documentatie.
```
Comment (kbdharun) [inline pages.nl/common/linode-cli.md:4]:
```suggestion
> Sommige subcommando's zoals `events` hebben een eigen documentatie pagina..
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md

**Review by kbdharun — APPROVED** [review]
The page additions LGTM. I noticed some issues with Windows alias pages and subcommand mention line overall.

- The page descriptions in Windows alias pages isn't in sync with the English page, especially the command name. Please update it. (I believe you ran the alias page script here so that might be the reason why this issue was caused)

File: pages.nl/windows/sl.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# sl
+
+> Dit commando is een alias van `set-location`.
```
Comment (kbdharun) [inline pages.nl/windows/sl.md:3]:
```suggestion
> In PowerShell is dit commando een alias van `Set-Location`.
```

File: pages.nl/windows/sc-query.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# sc-query
+
+> Dit commando is een alias van `sc`.
```
Comment (kbdharun) [inline pages.nl/windows/sc-query.md:3]:
```suggestion
> Dit commando is een alias van `sc.exe query`.
```

File: pages.nl/windows/sc-config.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# sc-config
+
+> Dit commando is een alias van `sc`.
```
Comment (kbdharun) [inline pages.nl/windows/sc-config.md:3]:
```suggestion
> Dit commando is een alias van `sc.exe config`.
```

File: pages.nl/windows/sc-create.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# sc-create
+
+> Dit commando is een alias van `sc`.
```
Comment (kbdharun) [inline pages.nl/windows/sc-create.md:3]:
```suggestion
> Dit commando is een alias van `sc.exe create`.
```

File: pages.nl/windows/sc-delete.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+# sc-delete
+
+> Dit commando is een alias van `sc`.
```
Comment (kbdharun) [inline pages.nl/windows/sc-delete.md:3]:
```suggestion
> Dit commando is een alias van `sc.exe delete`.
```

**Review by kbdharun — APPROVED** [review]

**Pull request merged**